### PR TITLE
Build fluent-bit-logzio image

### PR DESCRIPTION
**Pull request by duboisph**
# Description

This builds a custom fluent-bit image which includes the logz.io output plugin.

# Related Issue

https://github.com/skyscrapers/woorank/issues/77

# Motivation and Context

Shipping logs logz.io and all standard supported outputs.

# How Has This Been Tested?

```docker build .```

# Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist

- [x] I have followed the PR process as described [here](https://github.com/skyscrapers/documentation/blob/master/coding_guidelines/git.md#pull-requests)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by phonoloop — APPROVED** [review]
Looks good.

File: fluent-bit-logzio/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+RUN apk add --update curl && \
+  mkdir -p /fluent-bit/plugins && \
+  curl -L -o "/out_logzio.so" ${LOGZIO_SRC} 
+
+## FLUENT-BIT
+
+FROM fluent/fluent-bit:${FLB_VERSION}
+
+COPY plugins.conf /fluent-bit/etc/plugins.conf
```
Comment (iuriaranda) [inline fluent-bit-logzio/Dockerfile:17]: There might be some plugins already configured there, no?

Comment (duboisph) [reply fluent-bit-logzio/Dockerfile:17]: No. I double checked, the original file is empty/commented and no plugins are included in the upstream image.

Comment (duboisph) [reply fluent-bit-logzio/Dockerfile:17]: I double checked, the original one is empty/commented.

**Review by iuriaranda — APPROVED** [review]

**Pull request merged**